tests/unit/testwebsockethandler.cpp

```diff
@@ -173,7 +347,7 @@ void TestWebSocketHandler::tst_reconnectionAttemptsOnPingTimeout() {
   QSignalSpy socketDisconnectedSpy(&server, SIGNAL(socketDisconnected()));
 
   WebSocketHandler handler;
-  handler.testOverrideRetryInterval(100);
+  handler.testOverrideBaseRetryInterval(100);
```

Comment:
You can move the `ExponentialBackoffStrategy` object in a separate class and test it independently.

Reply:
All good! you are testing the exponentialBackoffstrategy already. I was suggesting to move it in a seprate class "file" (I forgot to write this word:) 
Ignore this comment.